core/templates/dev/head/pages/exploration_player/PlaythroughService.js

```diff
@@ -38,15 +39,21 @@ oppia.factory('PlaythroughService', [
       ISSUE_TYPE_CYCLIC_STATE_TRANSITIONS, EARLY_QUIT_THRESHOLD_IN_SECS,
       NUM_INCORRECT_ANSWERS_THRESHOLD, NUM_REPEATED_CYCLES_THRESHOLD,
       CURRENT_ACTION_SCHEMA_VERSION, ACTION_TYPE_EXPLORATION_START,
-      ACTION_TYPE_ANSWER_SUBMIT, ACTION_TYPE_EXPLORATION_QUIT) {
+      ACTION_TYPE_ANSWER_SUBMIT, ACTION_TYPE_EXPLORATION_QUIT,
+      RECORD_PLAYTHROUGH_PROBABILITY) {
     var playthrough = null;
     var expStopwatch = null;
+    var isPlayerInSamplePopulation = null;
 
     var multipleIncorrectStateName = {};
 
     var cycleIdentifier = {};
     var visitedStates = [];
 
+    var _isPlayerExcludedFromSamplePopulation = function() {
```

Comment:
Why is this private? Shouldn't this be a public function?

Reply:
@brianrodri Being private is enough right? The client services that PlaythroughServices is exposed to, doesnt need to know this information. This information is only used for computations and processing inside of PlaythroughService.

Reply:
Done.